kratos/tests/cpp_tests/containers/test_variables_list.cpp

```diff
@@ -58,5 +59,66 @@ KRATOS_TEST_CASE_IN_SUITE(VariablesListGetDofInfo, KratosCoreFastSuite) {
 
 }
 
+
+    class SetModelPartVariableToOne{
+            ModelPart& mrModelPart;
+        public:
+            SetModelPartVariableToOne(ModelPart& TheModelPart) : mrModelPart(TheModelPart){}
+
+            template<typename TVariableType> 
+            void operator()(TVariableType const& TheVariable) const {
+                for (auto &node : mrModelPart.Nodes())
```

Comment:
I am confused. I thought we consider K&R: https://en.wikipedia.org/wiki/Indentation_style#K&R_style